docs/DeveloperGuide.md

```diff
@@ -341,25 +341,25 @@ The following activity diagram summarizes what happens when a user enters an `ed
 In GuestBook, the user can mark all the rooms as unclean. This feature was added to make it easier for the hotel to transit to a new working day, as they would usually have to clean all the rooms when a new day starts.
 
 <span style="font-size: large; color: #e46c0a">Implementation:</span>
-* The `markroomsunclean` command edits all the guests in GuestBook and changes their isRoomClean statuses to "no". It takes in no additional inputs or fields.
+* The `markroomsunclean` command edits all the guests in GuestBook and changes their `isRoomClean` statuses to `no`. It takes in no additional inputs or fields.
 
 The following activity diagram summarises what happens when a user enters a `markroomsunclean` command.
 
 ![MarkRoomsUncleanActivityDiagram](images/MarkRoomsUncleanActivityDiagram.png)
-* When the user executes the `markroomsunclean` command, GuestBook will retrieve the list of all the guests that requires editing of the isRoomClean status.
-* A new list of guests will be created with isRoomClean fields set to "no". The other details of the guests will remain unchanged.
+* When the user executes the `markroomsunclean` command, GuestBook will retrieve the list of all the guests that requires editing of the `isRoomClean` status.
+* A new list of guests will be created with `isRoomClean` fields set to `no`. The other details of the guests will remain unchanged.
 * Finally, the new list of guests is set to the model.
 
 <span style="font-size: large; color: #e46c0a">Design Considerations:</span>
-**Aspect: The scope at which the command changes all guests' isRoomClean statuses**
+**Aspect: The scope at which the command changes all guests' `isRoomClean` statuses**
 * Alternative 1: Allow `markroomsunclean` command to operate only on the last shown list instead of the entire list. This is to standardise how edits are made across the commands (e.g. edit and delete).
```

Comment:
Should we use code blocks for `edit` and `delete`?

Reply:
Added!